### DISCUSSION TITLE
[Grades][Hw4 Hw1]

### DISCUSSION THREAD

**Discussion by kopter0** [prosyslab-classroom/cs348-information-security#739] [Q&A]
Second question: When claim for hw1 gonna happen, it still has '-' as entry

Solved:
First question: After regrading my grade became 0, and feedback disappeared, is it gonna be updated or is it final?

**Comment by KAIST-JongchanPark** [comment]
In grade, average score of Hw4 is 0.0. I think TA sets all students' score as 0 before announcing regraded score.

**Comment by Junseon00** [comment]
1. Yes, I set every score to 0, at that time and you can check your scores on KLMS now.
2. I'll ask this issue to TA in charge.

**Comment by dinmukhamedm** [comment]
Dear @junseon0-0 any updates on the second question?
